HackIllinois/ViewControllers/HIGroupPopupViewController.swift

```diff
@@ -0,0 +1,166 @@
+//
+//  HIGroupPopupViewController.swift
+//  HackIllinois
+//
+//  Created by HackIllinois Team on 2/1/21.
+//  Copyright © 2021 HackIllinois. All rights reserved.
+//  This file is part of the Hackillinois iOS App.
+//  The Hackillinois iOS App is open source software, released under the University of
+//  Illinois/NCSA Open Source License. You should have received a copy of
+//  this license in a file with the distribution.
+//
+
+import Foundation
+import UIKit
+import HIAPI
+
+protocol HIGroupPopupViewDelegate: class {
+    func updateInterests(_ groupPopupCell: HIGroupPopupCell)
+}
+
+class HIGroupPopupViewController: UIViewController {
+    // MARK: Properties
+    weak var delegate: HIGroupPopupViewDelegate?
+    let interests = ["AWS", "C++", "Project Management", "Python", "Docker", "Java", "ML", "Swift", "Go", "Javascript", "C", "Typescript"]
```

Comment:
Add a TODO comment, interests taken from API https://api.hackillinois.org/upload/blobstore/possible-interests/ in future PR